actors/multisig/src/types.rs

```diff
@@ -15,7 +15,7 @@ use serde::{Deserialize, Serialize};
 
 /// SignersMax is the maximum number of signers allowed in a multisig. If more
 /// are required, please use a combining tree of multisigs.
-pub(super) const SIGNERS_MAX: usize = 256;
+pub const SIGNERS_MAX: usize = 256;
```

Comment:
I tried to export this only to the testing package but couldn't figure it out so I made it fully public (I think?).  I don't think this is a problem, specs-actors has this fully public.  But if there's a better convention please tell me how to implement.

Reply:
There's no real problem with making this fully public. But this is a reason to move tests into the crate itself.

Reply:
lets keep them outside, it will be a lot of stuff.